tests/countries/test_sao_tome_and_principe.py

```diff
@@ -0,0 +1,326 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see AUTHORS.md file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from datetime import date
+from unittest import TestCase
+
+from holidays.countries.sao_tome_and_principe import SaoTomeAndPrincipe, ST, STP
+from tests.common import CommonCountryTests
+
+
+class TestSaoTome(CommonCountryTests, TestCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass(SaoTomeAndPrincipe)
+        cls.observed_holidays = SaoTomeAndPrincipe(observed=True)
+
+    def test_country_aliases(self):
+        self.assertIsInstance(ST(), SaoTomeAndPrincipe)
+        self.assertIsInstance(STP(), SaoTomeAndPrincipe)
+        self.assertEqual(ST().subdiv, None)
+        self.assertEqual(STP().subdiv, None)
+
+    def test_consistent_holidays(self):
+        # Test holidays that exist every year in the same form
```

Comment:
Please consider switching to the standardized test setups - [test_macau.py](https://github.com/vacanza/holidays/blob/ad51a0432db564ccae9a8918e1c57745aedc061c/tests/countries/test_macau.py#L20) and [test_suriname.py](https://github.com/vacanza/holidays/blob/ad51a0432db564ccae9a8918e1c57745aedc061c/tests/countries/test_suriname.py#L19) are some of the recent examples that you can use as references. 👍